packages/dds/tree/src/test/feature-libraries/editable-tree/editableTreeContext.spec.ts

```diff
@@ -34,13 +34,14 @@ async function createSharedTrees(
 describe("editable-tree context", () => {
     it("can't synchronize trees after the context been freed", async () => {
         const [provider, [tree1, tree2]] = await createSharedTrees(fullSchemaData, [personData], 2);
-        const person1 = tree1.root as Person;
-        const person2 = tree2.root as Person;
-        tree2.context.free();
 
+        const person1 = tree1.root as Person;
         assert.equal(person1.age, 35);
         person1.age = brand(42);
         assert.equal(person1.age, 42);
+
+        tree2.context.free();
```

Comment:
I'm confused with whats happening here.

tree2 (like all shared trees) owns its context, so the context should not be freed (unless the tree itself frees it).

This test then imminently uses tree2.root, which should depend on the context, but it doesn't error.

I'm also a bit confused what this test is trying to test. If you put shared tree in an invalid state by freeing the context its using, then we expect things to break, but I don't see why we would want a test for the specific way in which it breaks: having such a test doesn't seem useful unless the API actually defines whats supposed to happen in this case, and we want to test that that happens as specified.